src/demo/config.js

```diff
@@ -5,10 +5,10 @@
  */
 
 __CONFIG__ = {
-	url: 'http://localhost:8080',
 	basePath: '',
-	namespace: 'Liferay',
-	explainResolutions: true,
 	combine: false,
+	explainResolutions: true,
+	namespace: 'Liferay',
+	url: 'http://localhost:8080',
```

Comment:
Most of the changes are like this: ordering changes required by the "sort-keys" rule. Manual fixes, but relatively simple ones so hopefully not too error prone.